mobile/screens/AddTrainingLog.tsx

```diff
@@ -1,18 +1,89 @@
 import { StatusBar } from "expo-status-bar";
-import React, { useEffect } from "react";
-import { BackHandler, StyleSheet, Text, View } from "react-native";
+import React, { useEffect, useState } from "react";
+import {
+  BackHandler,
+  FlatList,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from "react-native";
+import HeaderDate from "../components/HeaderDate";
 
 export default function AddTrainingLogScreen(props: any) {
-  useEffect(() => {
-    BackHandler.addEventListener("hardwareBackPress", function () {
-      props.navigation.navigate("User Dashboard");
-      return true;
-    });
-  }, []);
+  const [totalHours, setTotalHours] = useState("");
+  const [skills, setSkills] = useState([
+    { id: 1, skill: "post", selected: "false" },
+    { id: 2, skill: "heel", selected: "false" },
+    { id: 3, skill: "jump", selected: "false" },
+  ]);
+  const [error, setError] = useState("");
+  const [skillsPlayed, setSkillPlayed] = useState("");
```

Comment:
they should be able to choose multiple -- make it so where this is a list of skills and selecting adds that skill to the list or deselecting removes that skill from the list